docs/keyboard-shortcuts.md

```diff
@@ -0,0 +1,59 @@
+# Gemini CLI Keyboard Shortcuts
+
+This document lists the available keyboard shortcuts in the Gemini CLI.
+
+## General
+
+| Shortcut | Description                                                                                                            |
+| -------- | ---------------------------------------------------------------------------------------------------------------------- |
+| `Ctrl+C` | Exit the application. Press twice to confirm. If the input prompt is not empty, the first press will clear the prompt. |
```

Comment:
Nit:
Maybe keep
Ctrl+C (1x)
and
Ctrl+C (2x) 
separate or move Ctrl+C (1x) down to the input prompt list.